src/inputs/RadioGroupField.tsx

```diff
@@ -155,7 +155,7 @@ function Radio<K extends string>(props: {
         </div>
         {description && (
           <div id={descriptionId} css={Css.sm.gray700.if(disabled).gray400.$}>
-            {description}
+            {typeof description === "function" ? description() : description}
```

Comment:
**thought:** Huh, surprised we allowed the description to be a function that returns a ReactNode... I wonder what this use case was? 🤔